doc/introduction.rst

```diff
@@ -148,8 +166,9 @@ a command prompt::
 
 .. warning::
 
-   Note that this is a "shell" command, that you need to type in a
-   command prompt, and not a Python command.
+   Note that this is a "shell" command, not a Python command. Concretely,
+   Mac users need to type this in the "Terminal", while Windows users
+   need to type it in command line of cmd.exe.
```

Comment:
`cmd.exe` send me back 6 or 7 years ago. We can cite it but I think that powershell should be evoked first.